docs/ChangeLog/20191129/PR7568.md

```diff
@@ -0,0 +1,5 @@
+# Encoder flip
```

Comment:
Would you please move/rename this file to `docs/ChangeLog/20200229/PR7568.md`?

Do that and I'll merge this into `future` branch.